library/coqlib.mli

```diff
@@ -208,4 +208,5 @@ val coq_iff_ref : global_reference lazy_t
 
 (* Deprecated functions *)
 val gen_reference :  message -> string list -> string -> global_reference
-[@@ocaml.deprecated "Please use Coqlib.find_reference"]
+(* Cannot mark as such here, mark should go in API.ml      *)
```

Comment:
Out of curiosity, how is this change related to OCaml 4.06.0?

Reply:
Better propagation of warning in module aliases.